src/libs/OptionsListUtils.ts

```diff
@@ -450,14 +450,14 @@ function getAllReportErrors(report: OnyxEntry<Report>, reportActions: OnyxEntry<
         const transactionID = parentReportAction?.actionName === CONST.REPORT.ACTIONS.TYPE.IOU ? parentReportAction?.originalMessage?.IOUTransactionID : null;
         const transaction = allTransactions?.[`${ONYXKEYS.COLLECTION.TRANSACTION}${transactionID}`];
         if (TransactionUtils.hasMissingSmartscanFields(transaction ?? null) && !ReportUtils.isSettled(transaction?.reportID)) {
-            reportActionErrors = {...reportActionErrors, smartscan: ErrorUtils.getMicroSecondOnyxError('report.genericSmartscanFailureMessage')};
+            reportActionErrors.smartscan = ErrorUtils.getMicroSecondOnyxError('report.genericSmartscanFailureMessage') as string;
```

Comment:
This looks wrong, `getMicroSecondOnyxError` returns an object, but you are asserting it to `string`.

Reply:
Btw, I think I [updated](https://github.com/Expensify/App/pull/34092/files#diff-ce1112b40e5691cce1b7697df1fef0e11b72948a2ae4f5cd910f003d231557f2) ErrorUtils types it in my PR (but since that PR wasn't reviewed yet, not sure if it makes sense to apply it)

Reply:
also actually this is exactly what is happening now on main 
<img width="1136" alt="image" src="https://github.com/Expensify/App/assets/25230417/953cf5ed-01d1-452c-8e79-74307d5043f5">

Reply:
So maybe `smartscan` is not a `string` then, it would be an `Errors` object